viadot/utils.py

```diff
@@ -117,3 +126,55 @@ def get_flow_last_run_date(flow_name: str) -> str:
         pendulum.parse(last_run_date_raw_format).format("YYYY-MM-DDTHH:MM:SS") + "Z"
     )
     return last_run_date
+
+
+@curry
+def custom_state_handler(
+    tracked_obj: Union["Flow", "Task"],
+    old_state: "prefect.engine.state.State",
+    new_state: "prefect.engine.state.State",
+    only_states: list = None,
+    API_KEY: str = None,
+) -> "prefect.engine.state.State":
```

Comment:
Whether the return type `"prefect.engine.state.State"` is a string or the type `prefect.engine.state.State`.